app/models/work_packages/costs.rb

```diff
@@ -31,7 +31,7 @@ module WorkPackages::Costs
 
   included do
     belongs_to :budget, inverse_of: :work_packages, optional: true
-    has_many :cost_entries, dependent: :delete_all
+    has_many :cost_entries, dependent: :delete_all, inverse_of: :entity
```

Comment:
"use `dependent: :destroy` or similar on WorkPackage.cost_entries - associated model CostEntry has callbacks that are currently skipped"

```suggestion
    has_many :cost_entries, dependent: :destroy, inverse_of: :entity
```